docs/iot/iot-partners/thingspark.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with thingspark Cloud
+
+The thingspark service is a cloud-based IoT solution that can be connected with various things. 
+
+Tizen developers can easily develop applications that can send IoT data to the cloud in real-time.
+
+Real-time data monitoring between the Tizen service app and the thingspark cloud is empowered through the thingspark plugin.
+
+The following diagram shows the interface between Tizen IoT device and thingspark cloud:
+
+
+**Figure: Architecture**
+
+![Architecture](media/architecture_thingspark.png)
+
+The thingspark plugin can be installed on the Tizen IoT devices that are compatible with the following:
+
+- Platform: Tizen 5.0 IoT Headed or Tizen 5.0 IoT Headless and higher
```

Comment:
Suggestion. 
How about changing this line to "Platform : Tizen 5.0 IoT Headed & Headless and higher"?

I worry the possibility of misreading as "Tizen 5.0 IoT Headed" or "Tizen 5.0 IoT Headless and higher", 
though "and higher" is for "Tizen 5.0 IoT Headed & Headless".

Reply:
@mijin85cho 
I completely agree with you.
My intention was to avoid "&". 
Can we rephrase to:
Tizen 5.0 IoT Headed and Headless, and higher

@xbldx 
Please verify and update.